### ISSUE TITLE
EIP-7840: blob counts configured at genesis

### ISSUE THREAD

**Issue by jflo** [hyperledger/besu#8026]
https://github.com/ethereum/EIPs/pull/9129

Implement a protocol schedule for blob targets and maximums.

We already had "targetBlobCount" as a config property on the prague genesis. Refactor that to allow a schedule per the EIP.

**Comment by daniellehrner** [comment]
@siladu Does it make sense to make the removal of EIP-7742 part of this issue? Or do you think is it better to track it in another one?

**Comment by siladu** [comment]
@daniellehrner Makes sense together I think as should be able to reuse some of the code hopefully. I actually think the 7742 code on main can remain inert: it would have only been triggered once populated via engine API which was never merged (https://github.com/hyperledger/besu/pull/7840)
